ming_sim/db.py

```
@@ -5080,7 +5080,7 @@ def drop_pending_actions_for_minister(self, turn: int, minister_name: str) -> in
     def save_resolve_context(
         self, turn: int, decree_text: str, narrative: str,
         simulator_payload: Dict[str, object],
-        secret_orders: Optional[List[Dict[str, object]]] = None,
+        secret_orders: Optional[Dict[str, object]] = None,  # #48：分组承载 {在办,待核议}（旧档为 list，json 落库不挑类型）
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)